ui/raidboss/data/06-ew/dungeon/the_tower_of_zot.txt

```diff
@@ -145,7 +145,7 @@ hideall "--sync--"
 2068.5 "Prapti Siddhi" sync / 1[56]:[^:]*:Sanduruva:62BB:/
 2071.9 "Manusya Thunder" sync / 1[56]:[^:]*:Minduruva:62C9:/
 
-2075.1 "--sync--" sync / 1[56]:[^:]*:Sanduruva:62A6:/ window 2075.1,30
+2075.1 "--sync--" sync / 1[56]:[^:]*:Sanduruva:62A6:/ window 100,30
```

Comment:
This happens in boss 2, so the sync needs to be smaller.

Reply:
Ah, so it does. And `test_timeline` would ignore this sync since the "correct" one takes precedence. Weird. I don't remember seeing it be a problem in-game, but it's definitely worth fixing!

Reply:
Oh, I caught this with test timeline!

Reply:
That's.... incredibly weird. I don't remember seeing that at all, it should have just immediately stumbled into a snarl of missed syncs, right? But I'm glad it worked!